### DISCUSSION TITLE
Data service

### DISCUSSION THREAD

**Discussion by ChenpiDog** [dataplane-app/dataplane#452] [Ideas & feature requests]
Provide data service API interface for application access.
Users only need to write SQL (or other simple ways). Dataplane generates relevant data service APIs for external applications to access data. For security, the data service APIs need to have corresponding API keys.

**Comment by saul-data** [comment]
I quite like this idea. Equally an UI interface to easily open up tables and parquet files to see and manage data.

**Reply by ChenpiDog** [reply]
The database can do permission control, but in practice, we will not directly provide the database link to the third party, but through the interface, from the interface to achieve data access permission control. Dataplane can replace data engineers to develop interfaces.

**Reply by saul-data** [reply]
This makes a lot of sense.
